experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -17,26 +17,14 @@ Returns the surface temperature of the earth, computed
 from the state u.
 """
 function get_land_temp_from_state(land_sim, u)
-    face_space = ClimaLSM.Domains.obtain_face_space(land_sim.domain.domain.subsurface.space)
-    N = ClimaCore.Spaces.nlevels(face_space)
-    interp_c2f = ClimaCore.Operators.InterpolateC2F(
-        top = ClimaCore.Operators.Extrapolate(),
-        bottom = ClimaCore.Operators.Extrapolate(),
-    )
-    surface_space = land_sim.domain.domain.surface.space
-    return ClimaCore.Fields.Field(
-        ClimaCore.Fields.field_values(
-            ClimaCore.Fields.level(interp_c2f.(u.bucket.T), ClimaCore.Utilities.PlusHalf(N - 1)),
-        ),
-        surface_space,
-    )
+    return ClimaLSM.surface_temperature(land_sim.model, u, land_sim.integrator.p)
```

Comment:
I think we may need to also pass an empty fourth argument to access this method? Im not sure. 

https://github.com/CliMA/ClimaLSM.jl/blob/507ec51e4032593b1cbe3ff10e916d8ee9f06e39/src/Bucket/bucket_parameterizations.jl#L36

Reply:
Good catch!